AmazonAutomotiveScenario.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
+namespace VectorIndexScenarioSuite
+{ 
+    internal class AmazonAutomotiveScenario : BigANNBinaryEmbeddingScearioBase
+    {
+        protected override string BaseDataFile => "base";
+        protected override string BinaryFileExt => "fbin";
+        protected override string QueryFile => "query";
+        protected override string GetGroundTruthFileName => "ground_truth";
+        protected override string PartitionKeyPath => "/id";
+        protected override string EmbeddingColumn => "embedding";
+        protected override string EmbeddingPath => $"/{EmbeddingColumn}";
+        protected override VectorDataType EmbeddingDataType => VectorDataType.Float32;
+        protected override DistanceFunction EmbeddingDistanceFunction => DistanceFunction.Euclidean;
+        protected override ulong EmbeddingDimensions => 384;
+        protected override int MaxPhysicalPartitionCount => 56;
+        protected override string RunName => "amazon-" + guid;
+
+        public AmazonAutomotiveScenario(IConfiguration configurations) : 
+            base(configurations, DefaultInitialAndFinalThroughput(configurations).Item1)
+        {
+        }
+
+        public override async Task Run()
+        {
+            await RunScenario();
+        }
+
+        public override void Setup()
+        {
+            this.ReplaceFinalThroughput(DefaultInitialAndFinalThroughput(this.Configurations).Item2);
+        }
+
+        private static (int, int) DefaultInitialAndFinalThroughput(IConfiguration configurations)
+        {
+            // default throughput for MSTuringEmbeddingOnlyScenario

```

Comment:
Remove reference to "MSTuringEmbeddingOnlyScenario". How big is this dataset? Do all sliceCount 100K, 1M, 10M and 100M even apply ? I thought it was a small dataset. Please remove slice counts that are not relevant.